cmd/expiration-mailer/main.go

```diff
@@ -126,10 +126,13 @@ func (m *mailer) sendNags(contacts []string, certs []*x509.Certificate) error {
 		ExpirationDate   string
 		DaysToExpiration int
 		DNSNames         string
+		DNSSerials       string
 	}{
 		ExpirationDate:   expDate.UTC().Format(time.RFC822Z),
 		DaysToExpiration: int(expiresIn.Hours() / 24),
 		DNSNames:         strings.Join(domains, "\n"),
+		DNSSerials: strings.Replace(
+			strings.Join(serials, "\n"), "<nil>", "", -1),
```

Comment:
Why is this replacement of `<nil>` required? I wouldn't expect there to be any `nil` certificate serials.